src/CodeXMLParse.cpp

```diff
@@ -1,7 +1,7 @@
 
 #include "CodeXMLParse.h"
 #include <r_util/r_annotated_code.h>
-
+#include <iostream>
```

Comment:
Is this needed?

Reply:
Not at all, I used it for printing something.